pyfar/dsp/dsp.py

```diff
@@ -1929,6 +1930,20 @@ def average(signal, mode='linear', caxis=None, weights=None, keepdims=False):
         If this is ``True``, the axes which are reduced during the averaging
         are kept as a dimension with size one. Otherwise, singular dimensions
         will be squeezed after averaging. The default is ``False``.
+    nan_policy: string, optional
+        Define how to handle NaNs in input signal.
+
+        ``'propagate'``
+           If the input signal includes NaNs, the corresponding averaged output
+           signal value will be NaN.
+        ``'omit'``
+           NaNs will be omitted while averaging. For each NaN value, the number
+           of values used for the average division is also reduced by one.
```

Comment:
I would suggest operation instead of division
```suggestion
           of values used for the average operation is also reduced by one.
```